doc/CONFIGURE.md

```diff
@@ -0,0 +1,205 @@
+
+This method does not make any updates to the Known Hosts file, instead using the file as a read-only source and expecting
+someone with suitable access to the appropriate user account on the Jenkins master to update the file as required,
+potentially using the ssh hostname command to initiate a connection and update the file appropriately.
+
+### Manually provided key Verification Strategy
+
+![](images/hkvs-manual-key.png)
+
+Checks the key provided by the remote host matches the key set by the user who configured this connection.
+
+The SSH key expected for this connection. This key should be in the form `algorithm value`
+where algorithm is one of ssh-rsa or ssh-dss, and value is the Base 64 encoded content of the key.
+
+### Manually trusted key Verification Strategy
+
+![](images/hkvs-trusted-key.png)
+
+Checks the remote key matches the key currently marked as trusted for this host.
+
+Depending on configuration, the key will be automatically trusted for the first connection, 
+or an authorised user will be asked to approve the key.
+An authorised user will be required to approve any new key that gets presented by the remote host.
+
+Require a user with Computer.CONFIGURE permission to authorise the key presented during the first connection to 
+this host before the connection will be allowed to be established.
+
+If this option is not enabled then the key presented on first connection for this host will be automatically trusted
+and allowed for all subsequent connections without any manual intervention.
+
+### Non verifying Verification Strategy
+
+![](images/hkvs-no-verify.png)
+
+Does not perform any verification of the SSH key presented by the remote host, allowing all connections regardless of the key they present.
+
+## Availability
+
+### Keep this agent online as much as possible
+In this mode, Jenkins will keep this agent online as much as possible.
+If the agent goes offline, e.g. due to a temporary network failure, Jenkins will periodically attempt to restart it.
+
+![](images/availability-as-much.png)
+
+### Take this agent online and offline at specific times
+In this mode, Jenkins will bring this agent online at the scheduled time(s), remaining online for a specified amount of time.
+If the agent goes offline while it is scheduled to be online, Jenkins will periodically attempt to restart it.
+
+After this agent has been online for the number of minutes specified in the Scheduled Uptime field, it will be taken offline. 
+If Keep online while builds are running is checked, and the agent is scheduled to be taken offline, 
+Jenkins will wait for any any builds that may be in progress to complete.
+
+![](images/availability-schedule.png)
+
+### Take this agent online when in demand, and offline when idle
+In this mode, Jenkins will bring this agent online if there is demand, i.e. there are queued builds which meet the following criteria:
+* They have been in the queue for at least the specified In demand delay time period
```

Comment:
You should either fix the capitalization of In or you should quote that or italicize it like *In demand delay time period*, especially if quoting the title of it.